lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -30,6 +30,59 @@ class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_5_9 {
 		'version',
 	);
 
+	/**
+	 * The valid properties under the settings key.
+	 *
+	 * @var array
+	 */
+	const VALID_SETTINGS = array(
+		'appearanceTools'       => null,
+		'prioritiseUserPresets' => null,
```

Comment:
Does this need to be added to the JSON [schema](https://github.com/WordPress/gutenberg/blob/848db705d6436bc267234910eb653968e6bf2893/schemas/json/theme.json) and/or added to the [theme.json docs](https://github.com/WordPress/gutenberg/blob/596afccfe1396584f11e7c5b7cc22bf1b24e4ced/docs/how-to-guides/themes/create-block-theme.md)?

Reply:
Also I think there are some `WP_Theme_JSON_Gutenberg_Test` E2E fails for output that no longer has `!important`